lib/handler/status.c

```diff
@@ -31,71 +38,48 @@ struct st_h2o_status_context_t {
     h2o_multithread_receiver_t receiver;
 };
 
+struct status_ctx {
+    int active;
+    void *ctx;
+};
 struct st_h2o_status_collector_t {
     struct {
         h2o_req_t *req;
         h2o_multithread_receiver_t *receiver;
     } src;
-    h2o_logconf_t *logconf;
     pthread_mutex_t mutex;
-    h2o_iovec_t data;
     size_t num_remaining_threads;
+    h2o_globalconf_t *gconf;
```

Comment:
I believe you do not need to retain a pointer to h2o_globalconf_t.  It can be reached by accessing `h2o_context_t::globalconf`.